### PR TITLE
Changed Dashboard to portrait format, rearranged widgets and added he…

### PR DESCRIPTION
**Pull request by vinayagg**
…lp text, updated title and documentation, updated screenshot

### What does this PR do?
Changed Dashboard to portrait format, rearranged widgets and added help text, updated title and documentation, updated screenshot

### Motivation
We updated wigets, help file, tile layout.

### Review checklist

- [x] PR has a [meaningful title](https://github.com/DataDog/integrations-core/blob/master/CONTRIBUTING.md#pull-request-title) or PR has the `no-changelog` label attached
- [x] Feature or bugfix has tests
- [x] Git history is clean
- [x] If PR impacts documentation, docs team has been notified or an issue has been opened on the [documentation repo](https://github.com/DataDog/documentation/issues/new)

### Additional Notes

Anything else we should know when reviewing?

### CODE REVIEW

**Comment by vinayagg** [comment]
@ChristineTChen @hithwen As per our various discussions. This is our final dashboard with various fixes and improvements. Please help get this past CI and merged. Thank you.

File: jfrog_platform/manifest.json [hunk 1]
```diff
@@ -24,7 +24,7 @@
   "integration_id": "jfrog-platform",
   "assets": {
     "dashboards": {
-      "Jfrog Artifactory Dashboard": "assets/dashboards/jfrog_artifactory_dashboard.json"
+      "JFrog Artifactory Dashboard": "assets/dashboards/jfrog_artifactory_dashboard.json"
```
Comment (ChristineTChen) [inline jfrog_platform/manifest.json:27]:
```suggestion
      "Jfrog Artifactory Dashboard": "assets/dashboards/jfrog_artifactory_dashboard.json"
```
This key cannot be changed (it's used for the backend) and has no impact on the dashboard title.

**Review by ChristineTChen — CHANGES_REQUESTED** [review]

File: jfrog_platform/README.md [hunk 2]
```diff
@@ -25,30 +25,21 @@ JFrog Artifactory's Datadog integration allows you to send Artifactory logs to t
 * Kubernetes Cluster
 * JFrog Artifactory and/or JFrog Xray installed via [JFrog Helm Charts][1]
 * [Helm 3][2]
+* Datadog's API key (you can get it from Datadog account -> Settings -> API)
```
Comment (ruthnaebeck) [inline jfrog_platform/README.md:28]:
```suggestion
* Your [Datadog API key][4].
```

File: jfrog_platform/README.md [hunk 3]
```diff
@@ -71,10 +62,16 @@ The API key is configured in `td-agent`, which will start sending logs to Datado
...
+### Step 4 Open JFrog Artifactory Dashboard
+Go to Dashboard -> Dashboard List, find `JFrog Artifactory Dashboard` and open it.
+
+### Step 5 Search for logs
+Logs can be access in the UI under  **Logs** > **Search**. Select the specific source mentioned in the Fluentd configuration (`fluentd` in configuration example above) to retrieve logs.
 
 
 [1]: https://github.com/jfrog/charts
 [2]: https://helm.sh/
 [3]: https://raw.githubusercontent.com/DataDog/integrations-extras/master/jfrog_platform/images/dashboard.png
```
Comment (ruthnaebeck) [inline jfrog_platform/README.md:77]:
```
[4]: https://app.datadoghq.com/account/settings#api
[5]: https://app.datadoghq.com/logs
```

File: jfrog_platform/README.md [hunk 4]
```diff
@@ -25,30 +25,21 @@ JFrog Artifactory's Datadog integration allows you to send Artifactory logs to t
 * Kubernetes Cluster
 * JFrog Artifactory and/or JFrog Xray installed via [JFrog Helm Charts][1]
 * [Helm 3][2]
+* Datadog's API key (you can get it from Datadog account -> Settings -> API)
 
-### Logs
+### Step 1 Fluentd Configuration
```
Comment (ruthnaebeck) [inline jfrog_platform/README.md:30]:
```suggestion
### Step 1 Fluentd configuration
```

File: jfrog_platform/README.md [hunk 5]
```diff
@@ -25,30 +25,21 @@ JFrog Artifactory's Datadog integration allows you to send Artifactory logs to t
...
-
-Once Datadog is setup, we can access logs in the UI under  **Logs** > **Search**. Select the specific source that to retrieve logs.
-
-If an API key exists, use the Datadog Fluentd plugin to forward logs directly from Fluentd to your Datadog account. 
-Adding proper metadata is the key to unlocking the full potential of your logs in Datadog. By default, the hostname and timestamp fields should be remapped.
-
-### Fluentd Configuration
-
-Integration is done by specifying the API key
+Fluentd integration is done by specifying the API key as follows:
```
Comment (ruthnaebeck) [inline jfrog_platform/README.md:33]:
```suggestion
Set up the Fluentd integration by specifying the API key as follows:
```

File: jfrog_platform/README.md [hunk 6]
```diff
@@ -25,30 +25,21 @@ JFrog Artifactory's Datadog integration allows you to send Artifactory logs to t
...
 
 _api_key_ is the API key from Datadog
```
Comment (ruthnaebeck) [inline jfrog_platform/README.md:35]:
```suggestion
_api_key_ is your [Datadog API key][4].
```

File: jfrog_platform/README.md [hunk 7]
```diff
@@ -59,9 +50,9 @@ _include_tag_key_ defaults to false and it will add fluentd tag in the JSON reco
 </match>
 ```
 
-## Datadog Demo
+### Step 2 Integration Enablement
```
Comment (ruthnaebeck) [inline jfrog_platform/README.md:53]:
```suggestion
### Step 2 Integration enablement
```

File: jfrog_platform/README.md [hunk 8]
```diff
@@ -59,9 +50,9 @@ _include_tag_key_ defaults to false and it will add fluentd tag in the JSON reco
...
 
-To enable this integration, run the td-agent on `artifactory` and `xray` pods
+To enable this integration, run the td-agent on `artifactory` pods
```
Comment (ruthnaebeck) [inline jfrog_platform/README.md:55]:
```suggestion
To enable this integration, run the td-agent on `artifactory` pods:
```

File: jfrog_platform/README.md [hunk 9]
```diff
@@ -71,10 +62,16 @@ The API key is configured in `td-agent`, which will start sending logs to Datado
 
 Add all attributes as facets from **Facets** > **Add** (on the left side of the screen in Logs) > **Search**.
 
-To access already existing visualizations and filters, click on Dashboards and add a new screenboard and then import [export.json][4] and overwrite the existing dashboard.
+### Step 3 Install JFrog Platform Tile
```
Comment (ruthnaebeck) [inline jfrog_platform/README.md:65]:
```suggestion
### Step 3 JFrog platform tile 
```

File: jfrog_platform/README.md [hunk 10]
```diff
@@ -71,10 +62,16 @@ The API key is configured in `td-agent`, which will start sending logs to Datado
 
 Add all attributes as facets from **Facets** > **Add** (on the left side of the screen in Logs) > **Search**.
 
-To access already existing visualizations and filters, click on Dashboards and add a new screenboard and then import [export.json][4] and overwrite the existing dashboard.
+### Step 3 Install JFrog Platform Tile 
+If you have not installed the JFrog Platform tile yet, install the tile.
```
Comment (ruthnaebeck) [inline jfrog_platform/README.md:66]:
```suggestion
If you have not installed the JFrog platform tile yet, install the tile.
```

File: jfrog_platform/README.md [hunk 11]
```diff
@@ -71,10 +62,16 @@ The API key is configured in `td-agent`, which will start sending logs to Datado
...
+
+### Step 4 Open JFrog Artifactory Dashboard
```
Comment (ruthnaebeck) [inline jfrog_platform/README.md:68]:
```suggestion
### Step 4 JFrog Artifactory Dashboard
```

File: jfrog_platform/README.md [hunk 12]
```diff
@@ -71,10 +62,16 @@ The API key is configured in `td-agent`, which will start sending logs to Datado
...
+Go to Dashboard -> Dashboard List, find `JFrog Artifactory Dashboard` and open it.
+
+### Step 5 Search for logs
+Logs can be access in the UI under  **Logs** > **Search**. Select the specific source mentioned in the Fluentd configuration (`fluentd` in configuration example above) to retrieve logs.
```
Comment (ruthnaebeck) [inline jfrog_platform/README.md:72]:
```suggestion
Access your [logs][5] in Datadog. Select the specific source mentioned in the Fluentd configuration (`fluentd` in configuration example above) to retrieve logs.
```

**Review by ruthnaebeck — CHANGES_REQUESTED** [review]
Docs review

File: jfrog_platform/README.md (hunk 3 above)
Comment (vinayagg) [reply jfrog_platform/README.md:77]: Hello there. I accepted all your changes. Only these two footer additions did not show up with "accept suggestion" button.

**Comment by vinayagg** [comment]
Hello @ChristineTChen and @ruthnaebeck . Thank you so much for your help. Is there anything else we can do to help merge this PR?

File: jfrog_platform/README.md (hunk 3 above)
Comment (ruthnaebeck) [reply jfrog_platform/README.md:77]: @vinayagg - You will need to add them using your text editor

Comment (vinayagg) [reply jfrog_platform/README.md:77]: @ruthnaebeck Thank you, I added these references.

**Comment by hithwen** [comment]
Hallo @vinayagg. It seems CI is failing because the dashboard json is not on the right format:
```
jfrog_platform...  FAILED
    /home/vsts/work/1/s/jfrog_platform/assets/dashboards/jfrog_artifactory_dashboard.json is not using the new /dashboard payload format.
````

**Review by ruthnaebeck — APPROVED** [review]
👍 for docs

File: jfrog_platform/assets/dashboards/jfrog_artifactory_dashboard.json [hunk 13]
```diff
@@ -1,661 +1,797 @@
 {
-    "author_name": "JFrog Platform",
+    "author_info": {
+        "author_name": "JFrog"
+    },
```
Comment (ChristineTChen) [inline jfrog_platform/assets/dashboards/jfrog_artifactory_dashboard.json:4]:
```suggestion
    "author_name": "JFrog",
```

I think you may have an old version of ddev. The export command was updated for this field to use `author_name` instead.

**Review by ChristineTChen — CHANGES_REQUESTED** [review]
LGTM, just left a nit to fix payload.

Minor nit: do you want to crop the white space out of the dashboard screenshot?

**Review by ChristineTChen — APPROVED** [review]

**Pull request merged**